### PR TITLE
Adding William Mercurio's profile to the website

### PR DESCRIPTION
**Pull request by wmercurio**

### CODE REVIEW

File: people/william-mercurio/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: bio
+datatype: bio
+
+title: William Mercurio
+picture: william-mercurio/profile.png
+abstract: "I am an undergraduate student in Dr. Caporaso's lab. I am currently working on some benchmarking studies looking at four different multiple sequence aligners with "Lane Mask" vs entropy based positional alignment filtering."
```
Comment (ebolyen) [inline people/william-mercurio/index.md]: Your quotes close early. Change this to a `>`

**Comment by ebolyen** [comment]
@gregcaporaso ready for merge

File: people/william-mercurio/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  I am an undergraduate student in Dr. Caporaso's lab. I am currently working on some benchmarking studies looking at four different multiple sequence aligners with "Lane Mask" vs entropy based positional alignment filtering.
+---
+
+
+# Background
+I am currently working towards a B.S. in Biology and Chemistry with minors in Astronomy, Mathematics, and Japanese.
+
+# Interests
+
+I am currently working on benchmarking studies looking at four different multiple sequence aligners (MUSCLE, MAFFT, PyNAST, and SSU-ALIGN). Each of these is being coupled with either "Lane mask" or strictly entropy based positional alignment filtering using SciKit-Bio.
```
Comment (gregcaporaso) [inline people/william-mercurio/index.md]: Please update *SciKit-Bio* to all lowercase (scikit-bio). This needs to be done in a couple of places in this text.

File: people/william-mercurio/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+We are looking to find out how the phylogenetic trees and diversity results com[are to the use of PyNAST multiple
```
Comment (gregcaporaso) [inline people/william-mercurio/index.md]: *com[are* -> *compare*

File: people/william-mercurio/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+sequence alignment with "Lane mask" positional alignment filtering. We are creating trees using FastTree, creating distance matrices
+using SciKit-Bio, and then performing Robinson Foulds tests on those trees using SciKit-Bio to get a correlation coefficient. We also seek to see how the different alignment and filtering methods effects the alpha and beta diversity.
```
Comment (gregcaporaso) [inline people/william-mercurio/index.md]: *Robinson Foulds* -> *Robinson-Foulds*. Also, this is a distance, not a correlation.

**Comment by gregcaporaso** [comment]
@wmercurio, a few comments on the text. I can merge this when you make those changes.

**Comment by wmercurio** [comment]
I made those corrections.

Thanks!

**Pull request merged**